RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,17 @@
+# Release procedure
+
+- Update [version_history.md](docs/root/version_history.md).
+  - Make sure breaking changes are explicitly called out.
+    Ideally these have been tracked on the go but it does not hurt to ask around and double
+    check. Running the python integration tests of the previous release with the latest binaries could serve as a double check for this.
+  - Ensure the release notes are complete. Ideally these have been tracked on the go.
+- Perform some thorough testing to double down on stability.
+
+```bash
+bazel test --cache_test_results=no --test_env=ENVOY_IP_TEST_VERSIONS=all --runs_per_test=1000 --jobs 50 -c dbg --local_resources 20000,20,0.25 //test:*
+```
+
+- Draft a [GitHub tagged release](https://github.com/envoyproxy/nighthawk/releases/new) using the [current version number on master]((include/nighthawk/common/version.h)). Earlier releases are tagged like `v0.1` so currently we use that as a naming convention.
```

Comment:
Curious - didn't we want to use semantic versioning (MAJOR.MINOR.PATCH)? I see that Envoy does use it.

(https://semver.org/spec/v2.0.0.html)

Reply:
Oh yeah good catch, starting the upcoming 0.3 that is the plan I guess, I’ll amend

Reply:
Amended